services/QuillLMS/app/controllers/teachers/classrooms_controller.rb

```diff
@@ -3,7 +3,7 @@ class Teachers::ClassroomsController < ApplicationController
   before_filter :teacher!
   # The excepted/only methods below are ones that should be accessible to coteachers.
   # TODO This authing could probably be refactored.
-  before_filter :authorize_owner!, except: [:scores, :units, :scorebook, :generate_login_pdf]
+  before_filter :authorize_owner!, only: [:update,  :transfer_ownership]
```

Comment:
If using `only` instead of `except` isn't already an explicit team policy, it probably should be.  I think we all do it now, but maybe it's worth finding the old lingering `except`s that might still be lying around and fixing them.